src/lib/y2storage/storage_manager.rb

```diff
@@ -59,7 +59,25 @@ class StorageManager
     # @return [Proposal, nil]
     attr_reader :proposal
 
-    def_delegators :@storage, :environment, :arch
+    def_delegators :@storage, :environment, :arch, :rootprefix
+
+    # @!method rootprefix
+    #   @return [String] root prefix used by libstorage
+
+    # Sets the root prefix used by libstorage in subsequent operations
+    #
+    # @param path [String]
+    def rootprefix=(path)
+      storage.rootprefix = path
+    end
```

Comment:
well, checking code of delegators it call `respond_to?` so if return false it is quite strange. and method is called with send, so I do not see much ways how it can failed. do you have exact failure which is caused? just for interest.